transit_gateway.tf

```diff
@@ -1,3 +1,11 @@
+data "ibm_is_vpc" "example" {
```

Comment:
Are the changes to transit_gateway.tf attempting to fix another unrelated issue? Is there more context?

Reply:
This logic was added as part of deploying management and workload vpc separately to create a fully private cluster during with public endpoint disable, but with the new base-ocp code which disables `confirm_network_healthy` when public endpoint is disabled, this change is deprecated. Will remove it.